sql_commands.txt

```diff
@@ -0,0 +1,59 @@
+***** Homework Submission
+
+1.
+SELECT *
+FROM players
+where team = 'NYK';
+
+2.
+SELECT *
+FROM players
+where team = 'IND'
+AND age < 26;
+
+3.
+SELECT *
+FROM players
+ORDER BY points ASC;
+
+4.
+SELECT name, (points/games)
+FROM players
+ORDER BY (points/games) DESC
+LIMIT 20
+
+5.
+SELECT avg(age)
+FROM players;
+
+6.
+SELECT avg(age)
+FROM players
+where team = 'OKC';
+
+7.
+SELECT avg(age)
+FROM players
+where games > 40;
+
+8.
+SELECT team, sum(points)
```

Comment:
You can give a name to derived columns with `as`:
`SELECT team, sum(points) as team_points`

Then that name can be used in the rest of this query. In the case of `ORDER BY` you can use the name instead of calling the SUM function again:
`ORDER BY team_points DESC;`